os/drivers/wireless/realtek/wifi_util_interface/rtk_drv.c

```diff
@@ -32,6 +32,7 @@
 
 #include <tinyara/lwnl/lwnl80211.h>
 #include <tinyara/lwnl/rtk_drv.h>
+#include <tinyara/kmalloc.h>
```

Comment:
1. This is not for this topic.
2. This was already merged. Could you rebase yours?

Reply:
updated